### ISSUE TITLE
[QUESTION]: Login form HTML

### ISSUE THREAD

**Issue by MESWEB** [justadudewhohacks/face-api.js#520]
It's possible to make a login form using face detection?

**Comment by abdulazizsapra** [comment]
yes possible. I've working on a asp.net project in which i used login with face id.

**Comment by MESWEB** [comment]
@abdulazizsapra Any tutorial?

**Comment by timsun28** [comment]
In case you want to create a basic implementation for a web app that uses face recognition follow this tutorial: https://youtu.be/AZ4PdALMqx0 (also linked on this repo) and start by creating a classifier that is able to recognize your face. 

Once you figured that out use a webcam as input instead of an image that you uploaded and link the result from the recognition to your existing login system for your app. 

Good luck!
